src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/indexes/+page.svelte

```diff
@@ -28,9 +28,9 @@
     let showDelete = false;
     let showCreateAttribute = false;
 
-    const handleDelete = async () => {
+    async function handleDelete() {
         invalidate(Dependencies.COLLECTION);
```

Comment:
What happens if the user clicks the button a bunch of times? Could it trigger an error?